test/suites/image_prefer_cached.sh

```diff
@@ -2,6 +2,7 @@
 # one is preferred, even if the its remote has a more recent one.
 test_image_prefer_cached() {
 
+  # shellcheck disable=SC1117
   if lxc image alias list | grep -q "^| testimage\s*|.*$"; then
```

Comment:
For all of those, I believe replacing "\s" by "\\s" should shut shellcheck up and so not require the warning be disabled.